lib/LTP/utils.pm

```diff
@@ -49,7 +49,7 @@ sub loadtest_runltp {
 
 sub shutdown_ltp {
     loadtest_kernel('proc_sys_dump') if get_var('PROC_SYS_DUMP');
-    loadtest_kernel('shutdown_ltp', @_);
+    loadtest_kernel('shutdown_ltp', @_) unless is_openstack;
```

Comment:
Can you add a comment why we don't execute this module in openstack?

Reply:
It is the same as for https://openqa.suse.de/tests/8363826. The test case failed due to it shutdown the instance, but openqa was still running a VM. Thus it has died. 
Nevertheless I will drop this part and focus on LTP in dedicated PR. Thanks!